### DISCUSSION TITLE
[ 2021년 06월 04일 (금) ] 피어세션 내용

### DISCUSSION THREAD

**Discussion by sglee487** [bcaitech1/p4-mod-model_diet#30] [General]
* automl 동시에 3개까지만 가능. 4개부터 거의 멈추더라.
* mobilenet v3 개조한거는 optimizer adamW, scheduler는 cosine restart 종류로, augment도 random이 오히려 더 안좋아서 sample aug가 더 좋다.
* 마지막 batch 불러올 때 버그성 비슷하게 이미지를 (512, 512, 3) 이런식으로 불러오는데 dataloader 매개변수를 last_drop=True로 하면 된다.
* optuna와 db 연결하는게 마스터세션 영상에 자세히 설명되어 있음.
* with_multiple을 0.2 로 조절하니 10,000,000에 f1 0.5 넘는거 같음.

## 주말에 어떤걸 할까
* 상건: 모듈에 따른 파라미터 수 파악하려고 함.
* 연세님: 자소서에 집중 ! (서버 엔지니어링..). 또 Daily Mission를 잘 못따라가서 도움 요청 (상건 이 도와주기로 함)
* 석진님: Daily Mission 을 다시 복습할 것 같다.
* 용우님: MACs를 10,000,000 으로 유지하면서 f1 0.5 를 넘겨보려고 함.
* 황훈님: 포트폴리오 완성, squeezenet 만들었는데 dropout, maxpooling generator 수정해야 하는데 에러없이 돌려보겠다.
* 웅희님: GPU에 한번에 다 올리는거 / 한번에 여러개 돌려보는거 하고싶음, invertedresidual v3랑 dwconv로만 모델 구성해서 optuna 돌려보거나 train 시키는거 해보려고 한다. 파라미터 조절해보면서 실험하겠다. 모델 학습에 집중할 것 같다.
